portal/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "oo7-portal"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+async-std = { version = "1.11", features = [ "attributes" ] }
+futures = "0.3"
```

Comment:
Use the specific crate needed instead of the whole futures, i would also double check the various used dependencies and check if we can disable some of the default features to avoid pulling useless deps

Reply:
We need FutureExt, I am not sure if that can be loaded without the entire futures crate.

Reply:
it comes from futures_util

Reply:
Done, thanks for the tip.